rc/definition.yaml

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
What is rc? if it is only for dummy/test data, maybe pick a more explicit name

Reply:
renamed as validate-meta